packages/components/form-control/src/form-control.tsx

```diff
@@ -411,12 +413,12 @@ export const Label = forwardRef<LabelProps, "label">(
 export type RequiredIndicatorProps = HTMLUIProps<"span">
 
 export const RequiredIndicator = forwardRef<RequiredIndicatorProps, "span">(
-  ({ className, ...rest }, ref) => {
+  ({ className, children, ...rest }, ref) => {
     const styles = useFormControlStyles() ?? {}
 
     const css: CSSUIObject = { ...styles.requiredIndicator }
 
-    return (
+    return typeof children !== "object" ? (
```

Comment:
Change `typeof children !== "object"` to `!isValidElement(children)` and add `import {  isValidElement } from "react"`.

Reply:
I fixed.